src/internal/pfsdb/pfsdb.go

```diff
@@ -208,43 +165,3 @@ func Branches(db *pachsql.DB, listener col.PostgresListener) col.PostgresCollect
 		}),
 	)
 }
-
-func ProjectKey(project *pfs.Project) string {
```

Comment:
Are these in `clusterstate` now?

Reply:
Yes they are, IIRC I don't think there will be any callers for the ProjectKey in the code base anymore and just the name will be used